bytecomp/emitcode.ml

```diff
@@ -168,6 +168,41 @@ and slot_for_c_prim name =
 let events = ref ([] : debug_event list)
 let debug_dirs = ref String.Set.empty
 
+(* Map from absolute paths of .ml files to their sanitized paths. *)
+let sanitize_map = Hashtbl.create 17
```

Comment:
This toplevel mutable structure probably need to be reset like we do for events.

Reply:
I have added code to clear sanitize_map, squashed, and pushed.